### ISSUE TITLE
Incorrect contrast ratio calculation due to incorrect font size

### ISSUE THREAD

**Issue by jducrot** [dequelabs/axe-core#293]
I have a link with a font size of 18px getting flagged incorrectly for not having high enough contrast ratio with its background. The problem is the calculated font size which axe is seeing at 13.5pt. As you can see here http://leaverou.github.io/contrast-ratio/#rgb%2839%2C%20124%2C%20173%29%20-on-rgb%28245%2C245%2C245%29 at 18px this shouldn't get flagged as a WCAG AA validation.

![addemail_console](https://cloud.githubusercontent.com/assets/280674/24179096/6ee9ba36-0e83-11e7-890f-398d9d8ae78e.jpg)

![add_email_axe](https://cloud.githubusercontent.com/assets/280674/24179104/76df4fda-0e83-11e7-9865-5a734d229e06.jpg)

**Comment by dylanb** [comment]
That is not calculated in pixels, it is calculated in points...however, from your screenshot, it does look larger than 13.5pt. Can you create a jsfiddle/codepen where the problem is reproduced?

**Comment by marcysutton** [comment]
@jducrot can you provide us with a demo? We'd like to get to the root cause, otherwise we'll close the issue for inactivity.

**Comment by jducrot** [comment]
@marcysutton This ticket can be closed. As @dylanb pointed out, I misread the font size requirement in the W3C specs. It says the contrast ratio requirements applies "for text that is at least 18 point (if not bold) or at least 14 point (if bold)". Since 18px is 13.5pt, axe is reporting the failure correctly.

**Comment by marcysutton** [comment]
Ah, thanks!
